CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Add `RSpec/LeakyConstantDeclaration` cop. ([@jonatas][], [@pirj][])
 * Improve `aggregate_failures` metadata detection of `RSpec/MultipleExpectations`. ([@pirj][])
 * Improve `RSpec/SubjectStub` detection and message. ([@pirj][])
+* Changed message of `RSpec/LetSetup` cop to be more descriptive. ([@foton][])
```

Comment:
Minor: probably change the tense to "Change".

Reply:
Done